go/upgrade/migrations/consensus_242.go

```diff
@@ -13,6 +13,8 @@ import (
 // This upgrade includes:
 //   - The `MayQuery“ field in the CHURP SGX policy, which defines which enclave identities
 //     are allowed to query runtime key shares.
+//   - An updated key manager policy update transaction that applies a new policy at the epoch
+//     boundary.
 const Consensus242 = "consensus242"
```

Comment:
Why is this still 242?

Reply:
Because it is bumped in https://github.com/oasisprotocol/oasis-core/pull/6289.